### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by kingqabil**
Used media queries to make sure that my website displays correctly on desktop screens.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -634,3 +633,390 @@ footer .footer-icons > li {
...
+    flex-direction: row;
+    align-items: flex-start !important;
+  }
+
+  .multi-post-container > img {
+    height: auto;
+    width: 55%;
+  }
+
+  .multi-post-content {
```
Comment (SajjadAhmad14) [inline style.css:761]: Please add an active image for the project section as it's broken. Kindly see the screenshot below for reference.
project

File: index.html [hunk 2]
```diff
@@ -226,6 +231,15 @@ <h2>I am a Software Developer</h2>
             </ul>
           </div>
         </div>
+        <div class="desktop-headline-bg-section d-flex">
```
Comment (SajjadAhmad14) [inline index.html:242]: Please make the icons clickable links. KIndly see the screenshot below for reference
icons
e.

File: index.html [hunk 3]
```diff
@@ -35,7 +35,12 @@
       <header class="d-flex font-inter">
```
Comment (SajjadAhmad14) [inline index.html:35]: Please add all the unnecessary files and folders like `.vscode` to `.gitignore`.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Sadiq Kayse, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
GitHub flow is followed. :heavy_check_mark:
README is professional. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello @Sadqhabil  great job working on the required changes 👍 💯

- [ ] [OPTIONAL] For the desktop session, you can change the links in the navbar to take you to their sections id instead of `#`

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**